k8s/zeppelin-server.yaml

```diff
@@ -213,14 +213,15 @@ rules:
   resources: ["roles", "rolebindings"]
   verbs: ["bind", "create", "get", "update", "patch", "list", "delete", "watch"]
 ---
-kind: RoleBinding
+kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: zeppelin-server-role-binding
 subjects:
 - kind: ServiceAccount
   name: zeppelin-server
+  namespace: default
```

Comment:
It is not sure that the `zeppelin-server` ServiceAccount is part of the `default` namespace.

Reply:
Yes, the namespace of zeppelin server is not sure. But the `clusterrolebinding` needs to specify the namespace of the service account, which is applied by user when user is creating the zeppelin server in k8s.

So if user does not apply the `zeppelin-server` in the default namespace, he also needs to modify this `namespace` field in the `clusterrolebinding`.

Reply:
Instead of creating a `ClusterRole` and a `ClusterRoleBinding`, I would prefer an additional blueprint with a `Role` and a `RoleBinding`. Inside the `RoleBinding` the user has to replace the `Namespace`.